lib/rules/no-console.js

```diff
@@ -17,6 +17,8 @@ const astUtils = require("../util/ast-utils");
 
 module.exports = {
     meta: {
+        type: "problem",
```

Comment:
I see that the meta.docs.category is set to "Possible Errors", but honestly I feel that this should be a "suggestion" (and maybe the category should be changed).

Reply:
Agree